nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-ui/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,61 @@
+nifi-transform-json-ui
```

Comment:
this should match the module nifi-scripting-ui

Reply:
Removed